spec/duration.html

```diff
@@ -2,44 +2,78 @@
 <meta charset="utf8" />
 
 <emu-clause id="sec-temporal-duration">
-    <h1>Duration Objects</h1>
-    <p>
-    Duration objects describe the difference between two like Date, Time, or DateTime objects.
-    They are only created via the <i>.minus()</i> methods of these objects.
-    </p>
+  <h1>Temporal.Duration Objects</h1>
+  <p>
+    A Temporal.Duration object describes the difference between two like Date,
+    Time, or DateTime objects. Objects of this type are only created via the
+    <i>.minus()</i> methods of these objects.
+  </p>
+  <ul>
+    <li>
+      The <i>.minus()</i> operation on Temporal.Date objects creates a
+      Temporal.Duration that only has years, months, and days properties.
+    </li>
+    <li>
+      The <i>.minus()</i> operation on Temporal.Time objects creates a
+      Temporal.Duration that only has hours, minutes, seconds, milliseconds,
+      microseconds, and nanoseconds properties.
+    </li>
+    <li>
+      The <i>.minus()</i> operation on Temporal.Instant objects creates a
+      Temporal.Duration like that of Time objects.
+    </li>
+    <li>
+      The <i>.minus()</i> operation on Temporal.DateTime objects creates a
+      Temporal.Duration that has all member properties, except if passed an
+      option indicating it should be created as a Time object or Date object.
+    </li>
+  </ul>
```

Comment:
I'm not sure there is consensus around this. Maybe add a note about that?

Reply:
These existed from before I started working on `Duration`, I kept it because I wasn't sure, but I'll add a note just to make sure.